en/docs/get-started/what-is-choreo.md

```diff
@@ -1,5 +1,13 @@
 # What is Choreo?
 
-!!! info
-    This section will introduce the Choreo platform, its basic capabilities, and its competitive advantage. It will also answer "Why Choreo?".
+Today, with the growth of the global API ecosystem, APIs have become an essential part of modern engineering. Therefore, it has become necessary for organizations to participate in this global API supply chain by consuming and producing APIs. The adoption of cloud and microservices-based distributed architecture has also become increasingly significant, causing more and more organizations to rapidly evolve and look for innovative ways to build their digital business ecosystem. This has led to the need for a state-of-the-art iPaaS that can address the complex challenges that organizations face when connecting applications, systems, and data to build integrations and expose those integrated services to internal and external consumers in a future-proof manner.
 
+Choreo, a versatile and comprehensive platform for low-code, cloud-native engineering, provides the following advantages:
+
+- An AI-assisted, low-code application development environment that can help developers build enterprise-class integrations faster while ensuring best practices and secure coding guidelines. 
+- Developers can switch between a visual low-code and typical code environment and select the experience that best supports a given situation.
+- Even without expert-level coding skills, Choreo's low-code environment empowers developers to build applications with ease. It also provides capabilities to collaborate and improve these applications as needed.
+- Its low-code application development platform auto-generates editable and version-controlled code and maintains it in GitHub. This enables applying the engineering rigor required for enterprise applications on low-code.
+- API management capabilities to develop APIs and get them to production. Choreo’s API Publisher Portal allows API developers to design, publish, and manage the lifecycle of APIs whereas the API Developer Portal allows consumers to consume APIs.
+- Realistic DevOps with versions, environments, CI/CD pipelines, and automation tests.
+- Deep observability to trace your executions from the code statement level to the machine it runs.
```

Comment:
Can we change "the machine it runs" to "the machine in which they run"?

Reply:
Changed this to "the machine on which those run"